docs/rp2/quickref.rst

```diff
@@ -92,11 +101,21 @@ See :ref:`machine.UART <machine.UART>`. ::
     uart1.write('hello')  # write 5 bytes
     uart1.read(5)         # read up to 5 bytes
 
+.. note::
+ 	
+	REPL over UART is disabled by default.You can see the :ref:`rp2_intro` for details on how to enable REPL over UART.
+
 
 PWM (pulse width modulation)
 ----------------------------
 
-How does PWM work on the RPi RP2xxx?
+
+There are 8 independent channels each of which have two outputs making it 16 PWM channels in total which can be clocked from 7Hz to 125Mhz. 
+
+
+The channels can be clocked by external sources, using an edge-sensitive input mode for frequency measurement, and a level-sensitive input mode for duty cycle measurement.
+
+For more information on this topic you can see the PWM module in the rp2040 datasheet.
```

Comment:
Add another link to it?
`datasheet <https://datasheets.raspberrypi.org/rp2040/rp2040-datasheet.pdf>`